### ISSUE TITLE
Corona reflections do not render on all roads

### ISSUE THREAD

**Issue by lopezloo** [multitheftauto/mtasa-blue#2750]
### Describe the bug

Some road models do not have `is_road` model flag set so these do not reflect [corona reflections](https://github.com/multitheftauto/mtasa-blue/pull/2347). This is single player issue.

### Steps to reproduce

```lua
crun setCoronaReflectionsEnabled(2) me.position = Vector3(2046, 1549, 10) marker = Marker(me.position, "corona", 1.0) marker:setCoronaReflectionEnabled(true)
```

### Describe the solution you'd like

We could set `is_road` flag for such models.

### Additional context

1. Depends on #1714.
2. There was single player mod fixing that but I can't find it now.

Example positions: large part of Las Venturas.

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not about a cheat or security vulnerability.

**Comment by FileEX** [comment]
> There was single player mod fixing that but I can't find it now.

Probably you mean this mod: https://www.gtainside.com/en/sanandreas/mods/70869-road-reflections-fix-1-0/

**Comment by lopezloo** [comment]
> > There was single player mod fixing that but I can't find it now.
> 
> Probably you mean this mod: https://www.gtainside.com/en/sanandreas/mods/70869-road-reflections-fix-1-0/

No, the issue this mod fixes was already fixed in MTA. There is mod which changes the `is_road` flag for models which have them not set.
